### PR TITLE
[CUBDIRMAN-96] Remove ambiguity for APPL_SERVER_PORT description

### PR DESCRIPTION
**Pull request by kisoo-han**
http://jira.cubrid.org/browse/CUBRIDMAN-96

for your temporary reference:
http://192.168.2.38/en/env.html#default-ports-for-cubrid
http://192.168.2.38/en/admin/config.html#broker-app-server-cas

### CODE REVIEW

File: ko/admin/config.rst [hunk 1]
```diff
@@ -2432,7 +2432,14 @@ CUBRID 설치 시 생성되는 기본 브로커 설정 파일인 **cubrid_broker
...
+    **APPL_SERVER_PORT** 는 Windows 운영체제에만 적용가능한 파라미터로, 클라이언트와 CAS의 통신 포트를 별도로 지정하기 위한 파라미터이다.
+
+    Linux 운영체제에서는 클라이언트가 브로커와 연결을 하면, 브로커는 CAS를 할당한 후 클라이언트와의 네트워크 연결을 그대로 CAS에게 전달하기 때문에 클라이언트는 재 접속 과정 없이 CAS와 통신이 가능하다. 이 경우, 별도의 **APPL_SERVER_PORT** 는 사용되지 않는다.
+
+    반면 Windows에서는 모든 CAS가 독립적인 네트워크 포트를 할당받아 네트워크 연결을 기다리고 있으며, 클라이언트가 브로커에 연결하면 브로커는 가용한 CAS로 접속하기 위한 포트 번호를 클라이언트에게 전달한다. 순차적으로 클라이언트는 현재의 브로커와 네트워크 연결을 종료하고 브로커에서 전달받은 포트 번호로 CAS와 새롭게 접속을 하게된다.
+
+    **APPL_SERVER_PORT** 를 별도로 설정하지 않으면 이 값은 **BROKER_PORT** 파라미터 값에 1을 더한 값이 된다.
+    예를 들어, **BROKER_PORT** 의 값이 30,000이고 **APPL_SERVER_PORT** 의 값을 설정하지 않은 상태에서 **MIN_NUM_APPL_SERVER** 의 값이 5이면 브로커 초기 구동 시 5개의 CAS가 각각 30,001~30,005의 포트를 사용한다. 같은 조건이고 **APPL_SERVER_PORT** 의 값만 35,000라면 브로커 초기 구동 시 5개의 CAS가 각각 35,000~35,004의 포트를 사용한다.  CAS의 최대 개수가 **cubrid_broker_conf** 의 **MAX_NUM_APPL_SERVER** 파라미터에 의해 제한되므로 설정할 수 있는 CAS의 통신 포트의 개수 역시 최대 **MAX_NUM_APPL_SERVER** 파라미터의 설정값으로 제한된다.
```
Comment (mhoh3963) [inline ko/admin/config.rst]:
```suggestion
    예를 들어, **BROKER_PORT** 의 값이 30,000이고 **APPL_SERVER_PORT** 의 값은 미설정 상태에서 **MIN_NUM_APPL_SERVER** 의 값이 5인 경우 브로커 초기 구동 시 5개의 CAS가 각각 30,001~30,005의 포트를 사용한다. 동일 조건 하에 **APPL_SERVER_PORT** 의 값만 35,000라면 브로커 초기 구동 시 5개의 CAS가 각각 35,000~35,004의 포트를 사용한다.  CAS의 최대 개수가 **cubrid_broker_conf** 의 **MAX_NUM_APPL_SERVER** 파라미터에 의해 제한되므로 설정할 수 있는 CAS의 통신 포트의 개수 역시 최대 **MAX_NUM_APPL_SERVER** 파라미터의 설정값으로 제한된다.
```

**Review by mhoh3963 — APPROVED** [review]

**Review by airnet73 — APPROVED** [review]

File: ko/admin/config.rst (hunk 1 above)
Comment (kisoo-han) [reply ko/admin/config.rst]: Thank you for your comment!

File: en/admin/config.rst [hunk 2]
```diff
@@ -2480,9 +2480,15 @@ Broker App. Server(CAS)
...
+    **APPL_SERVER_PORT** is a parameter to designate the TCP ports of CAS that communicates with application clients (**Windows only**).
 
-    On the Windows system, if a firewall exists between an application and a CUBRID broker, the communication port specified in **BROKER_PORT** and **APPL_SERVER_PORT** must be opened.
+    On Linux, established TCP connection between the BROKER and the client will be passed to the CAS. Therefore, the client can communicate with the CAS without additional TCP connection requests with the CAS.
```
Comment (charischau) [inline en/admin/config.rst]:
```suggestion
    On Linux, an established TCP connection between the BROKER and the client will be passed to the CAS. Therefore, the client can communicate with the CAS without additional TCP connection.
```
The 'with the CAS' at the end of the sentence seems like a redundancy, I would suggest removing it if the meaning of the sentence remains same.

File: en/admin/config.rst [hunk 3]
```diff
@@ -2480,9 +2480,15 @@ Broker App. Server(CAS)
...
+
+    On the other hand, in Windows, all available CASes are waiting for a connection on an independent TCP port, for example 33001/tcp, 33002/tcp. In this circumstances, when a client connects to a BROKER, the BROKER delivers the TCP port number, for example 33001/tcp, for connecting to an available CAS to the client. In sequence, the client terminates the current network connection with the BROKER and establishes a new connection with the CAS using the port number received from the BROKER.
```
Comment (charischau) [inline en/admin/config.rst]:
```suggestion
    On the other hand, in Windows, all available CASes are waiting for a connection on an independent TCP port, for example, 33001/tcp, 33002/tcp. In these circumstances, when a client connects to a BROKER, the BROKER delivers the TCP port number, for example, 33001/tcp, for connecting to an available CAS to the client. In sequence, the client terminates the current network connection with the BROKER and establishes a new connection with the CAS using the port number received from the BROKER.
```

File: en/admin/config.rst [hunk 4]
```diff
@@ -2480,9 +2480,15 @@ Broker App. Server(CAS)
...
+
+    If the **APPL_SERVER_PORT** parameter is not additionally specified, this value is the value obtained by adding 1 to the value of **BROKER_PORT**. For example, if the value of **BROKER_PORT** is 30,000 and the **APPL_SERVER_PORT** parameter has not been specified, and if the **MIN_NUM_APPL_SERVER** value is 5, then five CASes uses the TCP ports between 30,001 and 30,005, respectively. On the other hand, if the value of **APPL_SERVER_PORT** is 35,001 under the same conditions, 5 CAS uses TCP ports from 35,001 to 35,005. The maximum number of CAS specified in the **MAX_NUM_APPL_SERVER** parameter in **cubrid_broker_conf**; therefore, the maximum number of connection ports is also determined by the value of **MAX_NUM_APPL_SERVER** parameter.
```
Comment (charischau) [inline en/admin/config.rst]:
```suggestion
    If the **APPL_SERVER_PORT** parameter is not additionally specified, this value is the value obtained by adding 1 to the value of **BROKER_PORT**. For example, if the value of **BROKER_PORT** is 30,000 and the **APPL_SERVER_PORT** parameter has not been specified, and if the **MIN_NUM_APPL_SERVER** value is 5, then five CASes use the TCP ports between 30,001 and 30,005, respectively. On the other hand, if the value of **APPL_SERVER_PORT** is 35,001 under the same conditions, 5 CASes use TCP ports from 35,001 to 35,005. The maximum number of CAS specified in the **MAX_NUM_APPL_SERVER** parameter in **cubrid_broker_conf**; therefore, the maximum number of connection ports is also determined by the value of **MAX_NUM_APPL_SERVER** parameter.
```

File: en/env.rst [hunk 5]
```diff
@@ -281,7 +281,7 @@ The connection process between the application and the DB is as follows:
 #.  The cub_broker selects a connectable CAS.
 #.  The application and CAS are connected.
 
-    In Linux, BROKER_PORT, which is used as an application, is connected to CAS through the Unix domain socket. In Windows, since the Unix domain socket cannot be used, an application and CAS are connected through a port of which the number is the sum of the corresponding CAS ID and the APPL_SERVER_PORT value set in the cubrid_broker.conf. If the APPL_SERVER_PORT value has not been set, the port value connected to the first CAS is BROKER_PORT + 1.
+    In Linux, established TCP connection between the **BROKER** and the client will be passed to the **CAS**. Therefore, there is no need for an additional network port for the application to connect to the CAS. However, in **Windows**, when an application connects to a BROKER, the BROKER delivers the network port number to connect to the available CAS to the application. After the client closing the current connection with the BROKER, it connects to the CAS with the received network port number from the BROKER. If the **APPL_SERVER_PORT** parameter is not set, the network port used by the first CAS becomes BROKER_PORT + 1.
```
Comment (charischau) [inline en/env.rst]:
```suggestion
    In Linux, established TCP connection between the **BROKER** and the client will be passed to the **CAS**. Therefore, there is no need for an additional network port for the application to connect to the CAS. However, in **Windows**, when an application connects to a BROKER, the BROKER delivers the network port number to connect to the available CAS to the application. After the client closes the current connection with the BROKER, it connects to the CAS with the received network port number from the BROKER. If the **APPL_SERVER_PORT** parameter is not set, the network port used by the first CAS becomes BROKER_PORT + 1.
```

File: en/admin/config.rst (hunk 4 above)
Comment (mhoh3963) [inline en/admin/config.rst]:
```suggestion
    If the **APPL_SERVER_PORT** parameter is not additionally specified, this value is the value obtained by adding 1 to the value of **BROKER_PORT**. For example, if the value of **BROKER_PORT** is 30,000 and the **APPL_SERVER_PORT** parameter has not been specified, and if the **MIN_NUM_APPL_SERVER** value is 5, then five CASes uses the TCP ports between 30,001 and 30,005, respectively. On the other hand, if the value of **APPL_SERVER_PORT** is 35,000 under the same conditions, 5 CAS uses TCP ports from 35,000 to 35,004. The maximum number of CAS specified in the **MAX_NUM_APPL_SERVER** parameter in **cubrid_broker_conf**; therefore, the maximum number of connection ports is also determined by the value of **MAX_NUM_APPL_SERVER** parameter.
```
How about change it as like korean manual ?

**Review by charischau — APPROVED** [review]

File: en/admin/config.rst (hunk 2 above)
Comment (kisoo-han) [reply en/admin/config.rst]: You right, it may be a redundant. Thank you for your comment!

File: en/admin/config.rst (hunk 3 above)
Comment (kisoo-han) [reply en/admin/config.rst]: Thank you for your comment, I'll fix it.

File: en/admin/config.rst (hunk 4 above)
Comment (kisoo-han) [reply en/admin/config.rst]: You right, thank you for your comment!

File: en/env.rst (hunk 5 above)
Comment (kisoo-han) [reply en/env.rst]: Thank you for your comment!

File: en/admin/config.rst (hunk 4 above)
Comment (kisoo-han) [reply en/admin/config.rst]: In terms of consistency, your opinion is correct. Thank you for your comment!
I changed all port numbers examples with the Korean manual.

**Comment by kisoo-han** [comment]
@charischau , thanks for quick response!

**Review by airnet73 — APPROVED** [review]

**Pull request merged**